### PR TITLE
use more descriptive labels when marking PRs

### PR DESCRIPTION
**Pull request by Hydrox6**
I find it pretty weird when everything is a "plugin change", with no distinction as to whether that plugin is entirely new or not.

I also must apologise: I've not written TypeScript before, and I'm not sure if this is even valid, let alone the best way to do it, but I wanted to have a try. I'm also not sure how to test it at all, to be honest, so sorry if this is complete trash

### CODE REVIEW

**Comment by abextm** [comment]
setHasLabel needs to be called with false when the condition isn't true so it can remove the label

**Comment by Hydrox6** [comment]
so I would call them all with `false` before the `forEach` to remove the labels first?

**Comment by abextm** [comment]
no that would generate an event in the ui which would be stupid. Just do the same thing that all the other `setHasLabel`s do

File: index.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { Application, ProbotOctokit } from "probot";
 import { OctokitResponse, PullsListFilesResponseData } from "@octokit/types";
 type Octokit = InstanceType<typeof ProbotOctokit>;
 
+const NEW_PLUGIN = "new plugin";
+const REMOVE_PLUGIN = "remove plugin";
```
Comment (Nightfirecat) [inline index.ts:6]: Maybe "plugin (added|removed)"?

Comment (Hydrox6) [reply index.ts:6]: Good idea, would fit with the existing `plugin change`

**Pull request merged**